repos/spack_repo/builtin/packages/py_xarray/package.py

```diff
@@ -41,11 +42,17 @@ class PyXarray(PythonPackage):
     variant("io", default=False, description="Build io backends")
     variant("parallel", default=False, description="Build parallel backend")
     variant("viz", default=False, when="@2024.7.0:", description="Buid viz backends")
+    variant("etc", default=False, when="@2025.7.1:", description="Etc")
+    variant("accel", default=False, when="@2025.7.1:", description="Accerlators, e.g., numba")
 
     # pyproject.toml
     depends_on("py-setuptools", when="@:0.15", type="build")
     depends_on("py-setuptools@38.4:", when="@0.16:", type=("build", "run"))
     depends_on("py-setuptools@42:", when="@0.17:", type=("build", "run"))
+
+    # at least 2025.x is only build dep
```

Comment:
Seems like the run-time dependency on setuptools was removed in 0.20: https://github.com/pydata/xarray/pull/5845

Note that the older `depends_on` lack an upper bounds, and so setuptools is still marked as required at run-time.